From 0c95f457a321de6542f81e799425454cb31a78e4 Mon Sep 17 00:00:00 2001
From: Duy Bao Dang <duy.dang.yb@renesas.com>
Date: Fri, 22 Jan 2021 15:09:57 +0700
Subject: [PATCH 431/439] ARM: dts: r8a7742: Add VIN[0-3] aliases

This patch adds aliases for VIN[0-3] in the R8A7742 (RZ/G1H) SoC.
VIN aliases are used to specify which video input device will be used.

Signed-off-by: Duy Bao Dang <duy.dang.yb@renesas.com>
---
 arch/arm/boot/dts/r8a7742.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index b86eb373..1286a3f 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -20,6 +20,10 @@
 		i2c1 = &i2c1;
 		i2c2 = &i2c2;
 		i2c3 = &i2c3;
+		vin0 = &vin0;
+		vin1 = &vin1;
+		vin2 = &vin2;
+		vin3 = &vin3;
 	};
 
 	/*
-- 
2.7.4

